front/components/poke/plans/form.tsx

```diff
@@ -227,6 +228,7 @@ export const PLAN_FIELDS = {
       { value: "fixed", label: "Fixed" },
       { value: "free", label: "Free" },
       { value: "monthly_active_users", label: "MAU" },
+      { value: "monthly_active_seats", label: "Seats" },
```

Comment:
I'm happy to take separately, but I think we want to have `BILLING_TYPE_SELECT_CHOICES: Record<FreeBillingType | PaidBillingType, {value: string, label: string}>` to make it impossible to forget to update this select as we add/remove billing types.

Reply:
Done!